ZTL DataLoading action failing on "dataloading (dof_shoreline)"

Sending over to you guys immediately because we don't update this specific dataset, leading me to think it may be code related.

[Partial error message](https://github.com/NYCPlanning/data-engineering/actions/runs/16725465534/job/47339800032):
```
/__w/data-engineering/data-engineering/dcpy/lifecycle/ingest/transform.py:427: SyntaxWarning: invalid escape sequence '\d'
  pat: -[a-zA-Z\d]1$
Usage: python -m dcpy lifecycle ingest [OPTIONS] COMMAND [ARGS]...
Try 'python -m dcpy lifecycle ingest --help' for help.
╭─ Error ──────────────────────────────────────────────────────────────────────╮
│ No such command 'dof_shoreline'.                                             │
╰──────────────────────────────────────────────────────────────────────────────╯
Error: Process completed with exit code 2.
```

--- a/.github/workflows/zoningtaxlots_dataloading.yml
+++ b/.github/workflows/zoningtaxlots_dataloading.yml
@@ -36,11 +36,14 @@ jobs:
   dataloading:
     runs-on: ubuntu-22.04
     container:
-      image: nycplanning/build-base:${{ inputs.dev_image && format('dev-{0}', github.head_ref || github.ref_name) || 'latest' }}
+      image: nycplanning/build-base:${{ inputs.dev_image && format('dev-{0}', github.head_ref
+        || github.ref_name) || 'latest' }}
     env:
       TEMPLATE_DIR: ./ingest_templates
-      RECIPES_BUCKET: ${{ inputs.dev_bucket && format('de-dev-{0}', inputs.dev_bucket) || 'edm-recipes' }}
-      PUBLISHING_BUCKET: ${{ inputs.dev_bucket && format('de-dev-{0}', inputs.dev_bucket) || 'edm-publishing' }}
+      RECIPES_BUCKET: ${{ inputs.dev_bucket && format('de-dev-{0}', inputs.dev_bucket) ||
+        'edm-recipes' }}
+      PUBLISHING_BUCKET: ${{ inputs.dev_bucket && format('de-dev-{0}', inputs.dev_bucket) ||
+        'edm-publishing' }}
     strategy:
       matrix:
         dataset:
@@ -75,7 +78,7 @@ jobs:
 
     - name: Ingest
       run: |
-        python3 -m dcpy lifecycle ingest ${{ matrix.dataset }} \
+        python3 -m dcpy lifecycle ingest ingest ${{ matrix.dataset }} \
           --latest \
           --push \
           ${{ inputs.overwrite && '--overwrite' || '' }}

